package.json

```diff
@@ -1,28 +1,18 @@
 {
-  "name": "stoplight-elements-apis",
-  "description": "Basic React app with embedded Stoplight Elements: StoplightAPI and Stoplight Project",
-  "author": {
-    "name": "Michael Miaskowski",
-    "email": "mmiask92@gmail.com"
-  },
-  "version": "0.2.0",
+  "name": "@stoplight/elements-starter-react",
+  "version": "1.0.0",
   "private": true,
+  "description": "Basic React app with embedded Stoplight Elements: StoplightAPI and Stoplight Project",
```

Comment:
The API component is simply `API` not `StoplightAPI` because it's not tied to Stoplight. Also `StoplightAPI` is written without a space while `Stoplight Project` is with one.

Actually for maintenance concerns I would simply remove the part after the colon: We'll likely extend this example if we release other components and we'll absolutely forget to update this description.